packages/api/src/platforms/vtex/utils/skuVariants.ts

```diff
@@ -189,7 +191,7 @@ export function getFormattedVariations(
 
       const formattedVariant = {
         src: variantImageToUse.imageUrl,
-        alt: variantImageToUse.imageLabel ?? '',
+        alt: variantImageToUse.imageText ?? '',
```

Comment:
Is this the right field? This will change the actual behavior.

```suggestion
        alt: variantImageToUse.imageText ?? variantImageToUse.imageLabel ?? '',
```

Reply:
Maybe this shouldn't be here as well, but our old behavior was wrong. I can move it to a separate PR if you want.

Reply:
it's not a problem for me. let's go ahead